test/http2/plain.test.js

```diff
@@ -2,7 +2,7 @@
 
 const t = require('tap')
 const test = t.test
-const Fastify = require('../..')
+const Fastify = require('../../fastify')
```

Comment:
I don't think this change is needed.

Reply:
right, change done (here and in other places)